docker-compose.yml

```diff
@@ -50,6 +50,7 @@ services:
     volumes:
       - ./public:/usr/src/gogovsg/public
       - ./src:/usr/src/gogovsg/src
+      - ~/Projects/ContainerLogs:/root/.npm/_logs
```

Comment:
this might have been an accidental commit. looks like a volume mount you created to debug yesterday

Reply:
This is intentional actually, the folder will be created when we run the docker-compose, basically, it will help with our debugging moving forward.

Reply:
the `~/Projects/ContainerLogs` part seems specific to your computer, or am i misunderstanding this?

Reply:
on a second thought, I changed to ./.logs folder and updated the gitignore file. I think this looks better.